yt_dlp/extractor/youtube.py

```diff
@@ -4227,13 +4319,17 @@ def build_fragments(f):
                     f'{video_id}: Some formats are possibly damaged. They will be deprioritized', only_once=True)
 
             client_name = fmt[STREAMING_DATA_CLIENT_NAME]
-            po_token = fmt.get(STREAMING_DATA_PO_TOKEN)
+            po_token = fmt.get(STREAMING_DATA_INITIAL_PO_TOKEN)
```

Comment:
I'm a bit confused how we are adding the framework for a PO token fetcher but then we only use the initial PO token in the actual formats code

Reply:
Sneaking this in for future use when the downloader needs to refresh the po token (hint: sabr)